pkg/operator/rgw/rgw.go

```diff
@@ -89,8 +110,141 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+type idType struct {
+	ID string `json:"id"`
+}
+
+func (c *Cluster) createRealm(serviceIP string) error {
+	realmArg := fmt.Sprintf("--rgw-realm=%s", c.Config.Name)
+	zonegroupArg := fmt.Sprintf("--rgw-zonegroup=%s", c.Config.Name)
+	zoneArg := fmt.Sprintf("--rgw-zone=%s", c.Config.Name)
+	endpointArg := fmt.Sprintf("--endpoints=%s:%d", serviceIP, c.Config.Port)
+	updatePeriod := false
+
+	// create the realm if it doesn't exist yet
+	output, err := c.runRGWCommand("realm", "get", realmArg)
```

Comment:
The `runRGWCommand` function doesn't have an output file to parse only the output from, it parses both stdout and stderr (combined output), right?  Are we at risk of getting malformed output when debugging logging is on, similar to #942?

Reply:
good question, i'll confirm